src/traits/base.php

```diff
@@ -27,6 +27,13 @@
  */
 trait Base {
 
+	/**
+	 * Mock
+	 *
+	 * @var \PMC\Unit_Test\Mocks\Factory
+	 */
+	public $mock;
```

Comment:
This (and the accompanying `@var` doc) will help our IDE/editor PHP intelligence, such as Intelephense, (a) understand that `$this->mock` exists and (b) what type of object/methods/props are available.

I traced the entire inheritance tree of `\PMC\Unit_Test\Tests\Base` and only found `\PMC\Unit_Test\Mocks\Factory` referencing the existence of OR setting the value of a `mock` property. Since this is the only class involving it, we should actually declare it here so we don't run into errors down the road.